File: first/php/library/arl.php
<?php
    class Arl extends Discounts {
        private int $level;
        private float $arl;

        public function __construct(int $level) {
            $this->level = $level;
            if ($level>5 || $level<1){
                echo 'Risk Level non-existent<hr>';
                $this->arl = 0;
            }else{
                $this->setArl();
            }
        }
        
        private function setArl(): void {
            switch ($this->level) {
                case 1:
                    $this->arl = 0.005;
                    break;
                case 2:
                    $this->arl = 0.01;
                    break;
                case 3:
                    $this->arl = 0.02; 
                    break;
                case 4:
                    $this->arl = 0.04;
                    break;
                case 5:
                    $this->arl = 0.06;
                    break;
            }
        }
    
        public function discounts(int $salary): float {
            return $salary * $this->arl;
        }
        
    }?>

File: first/java/Arl.java

public class Arl extends Discounts {

    private final int level;
    private float arl;

    public Arl(int level) {
        this.level = level;

        if (level < 1 || level > 5) {
            System.out.println("Risk Level non-existent\n");
            this.arl = 0;
        } else {
            setArl();
        }
    }

    private void setArl() {
        switch (level) {
            case 1 -> arl = 0.005f;
            case 2 -> arl = 0.01f;
            case 3 -> arl = 0.02f;
            case 4 -> arl = 0.04f;
            case 5 -> arl = 0.06f;
        }
    }

    @Override
    public float discounts(int salary) {
        return salary * arl;
    }
}